### ISSUE TITLE
Add support .css files

### ISSUE THREAD

**Issue by kyoz** [ryanoasis/vim-devicons#259]
Seem like css files do use icon of C#. And C# file doesn't have icon.

I think It'd be awesome if it have it own icon, right? :smile: 

![image](https://user-images.githubusercontent.com/29815830/53082334-39be0400-352f-11e9-9a50-5bacbc23d28e.png)
![screenshot from 2019-02-21 09-22-34](https://user-images.githubusercontent.com/29815830/53138939-93b7db80-35ba-11e9-8e81-61a98d001026.png)

This is the css icon i'm using in vscode (with material icon theme). 
![image](https://user-images.githubusercontent.com/29815830/53082504-8dc8e880-352f-11e9-8ab4-5aad67a383b9.png)

**Comment by ryanoasis** [comment]
Funny but yeah the hash is a bit generic. You bring up good cases. Need to think on this one.

**Comment by kazukazuinaina** [comment]
@kyoz 

I wrote a patch that is added `.cs`'s file icon.

**Comment by kazukazuinaina** [comment]
@ryanoasis 

Thank you

**Comment by n0ibe** [comment]
Why has this issue been closed? The css icon is still a hash, having the css logo would be better.

**Comment by kazukazuinaina** [comment]
I've investigated this issue and the visual studio code actually uses the # icon. 
Also, if you don't like the # icon, what is the proper icon for css? I don't know. 
The lack of a c# icon was the more important issue in this issue, and it has been closed because it has been resolved.

> having the css logo would be better.

I don't think this can be done because the nerd font does not include the css logo. 
This is not a problem with vim-devicons.
